exercises/prime-factors/prime_factors.py

```diff
@@ -1,2 +1,2 @@
-def prime_factors():
+def prime_factors(number):
```

Comment:
`natural_number` might be a better feet according to https://github.com/exercism/python/blob/master/exercises/prime-factors/README.md
What do you think?

Reply:
@m-a-ge That looks like a better parameter name. I'll make the update immediately.